build.sbt

```diff
@@ -1,15 +1,22 @@
 import Dependencies._
 import org.scalafmt.sbt.ScalafmtPlugin.autoImport.scalafmtOnCompile
+import sbt.Keys.{semanticdbEnabled, semanticdbVersion}
 import sbt.VirtualAxis.ScalaVersionAxis
-import sbt.io.Path.userHome
-import sbt.librarymanagement.Patterns
 
 lazy val supportedScalaVersions = List(scalaVersion213, scalaVersion212)
 
 ThisBuild / scalaVersion := supportedScalaVersions.head
-ThisBuild / version := "0.1.0-SNAPSHOT"
 ThisBuild / organization := "org.virtuslab"
 ThisBuild / organizationName := "VirtusLab"
+ThisBuild / versionScheme := Some("early-semver")
```

Comment:
`early` as in 0.x.y rather than (n>=1).x.y?

Reply:
Enumeration of version schemes [here](https://www.scala-sbt.org/1.x/docs/Publishing.html#Version+scheme)